### PR TITLE
Show correct type of error message for CI flags

### PR DESCRIPTION
**Pull request by dmecke**
Hi there!

First of all, thank you very much for the great tool!

When using both CI flags, --min-msi and --min-covered-msi together and only min-covered-msi fails, it shows a message that the minimum required min-msi percentage is not reached which is wrong.

This pull request checks boths values individually and shows the corresponding error message.

Please tell me if I need to improve something in this pull request.

### CODE REVIEW

File: src/InfectionApplication.php [hunk 1]
```diff
@@ -248,6 +259,13 @@ private function getBadMsiErrorMessage(MetricsCalculator $metricsCalculator): st
             );
         }
 
+        throw new \LogicException('Seems like something is wrong with calculations and min-msi options.');
+    }
+
+    private function getBadCoveredMsiErrorMessage(MetricsCalculator $metricsCalculator): string
+    {
+        $baseMessage = 'The minimum required %s percentage should be %s%%, but actual is %s%%. Improve your tests!';
```
Comment (borNfreee) [inline src/InfectionApplication.php]: Could you please reduce code duplication a little bit:

1. This message probably can be moved to const and reused on line 251
2. and there is a duplicated exception message `throw new \LogicException('Seems like something is wrong with calculations and min-msi options.');`. I suggest to create a custom one with incapsulated message or reuse this message somehow in this class

**Comment by borNfreee** [comment]
Hello, thank you for using this tool and for this PR. Nice catch!

Left a small comment about duplicated strings, besides that I'm happy to merge it.

File: src/InfectionApplication.php (hunk 1 above)
Comment (dmecke) [reply src/InfectionApplication.php]: I've now extracted the `$baseMessage` into a constant and created a new `MsiCalculationException`.

**Comment by borNfreee** [comment]
Thank you @dmecke !

**Pull request merged**